src/views/Login.vue

```diff
@@ -48,11 +53,21 @@ export default defineComponent({
   data() {
     return {
       username: "",
-      password: ""
+      password: "",
+      instanceUrl: ""
     };
   },
+  computed: {
+    ...mapGetters({
+      instanceUrlSaved: 'user/getInstanceUrl'
+    })
+  },
+  mounted() {
+    this.instanceUrl= this.instanceUrlSaved;
+  },
   methods: {
     login: function () {
+      this.store.dispatch("user/setUserInstanceUrl", this.instanceUrl)
```

Comment:
Use await to avoid any side effects in rare cases

Reply:
This action is working synchronously, so we can avoid using async and await.
Thank You for reviewing my code.